frontend/scss/vendor/mathigon.scss

```diff
@@ -4,6 +4,115 @@
 @import '../variables/mq.scss';
 @import '../variables/fonts.scss';
 
+x-blank, x-blank-mc {
+  .target {
+    background: $background-color-lighter;
```

Comment:
would this still work if we specified `background-color`?

Reply:
yup, thanks... updates.